exercises/rna-transcription/test/test_rna_transcription.c

```diff
@@ -0,0 +1,45 @@
+#include "vendor/unity.h"
+#include "../src/rna_transcription.h"
+#include <stdlib.h>
+#include <string.h>
+
+void resetTest(void) {}
+
+void test_transcription(const char *dna, const char* expected) {
+  char *rna = to_rna(dna);
+  TEST_ASSERT_TRUE(strncmp(rna, expected, strlen(dna)) == 0);
+  free(rna);
+}
+
+void test_transcribes_G_to_C(void) {
+  test_transcription("G", "C");
+}
+
+void test_transcribes_C_to_G(void) {
+  test_transcription("C", "G");
+}
+
+void test_transcribes_T_to_A(void) {
+  test_transcription("T", "A");
+}
+
+void test_transcribes_A_to_U(void) {
+  test_transcription("A", "U");
+}
+
+void test_transcribes_all_occurrences(void) {
+  test_transcription("ACGTGGTCTTAA", "UGCACCAGAAUU");
+}
+
```

Comment:
There are some tests missing from the [common test suite for this exercise](https://github.com/exercism/x-common/blob/master/exercises/rna-transcription/canonical-data.json). Can you add the tests related to input validation?